consensus/src/lib.rs

```diff
@@ -208,13 +208,12 @@ impl<TYPES: NodeTypes> Consensus<TYPES> {
                 .get_leaf_commitment()
                 .ok_or_else(|| HotShotError::InvalidState {
                     context: format!(
-                        "Visited failed view {:?} leaf. Expected successfuil leaf",
-                        start_from
+                        "Visited failed view {start_from:?} leaf. Expected successfuil leaf"
```

Comment:
Nice! Didn't know this worked with the :? expansion.

Reply:
Yeah! Super cool.